src/test/java/dev/openfeature/sdk/OpenFeatureAPITest.java

```diff
@@ -40,6 +44,20 @@ void namedProviderOverwrittenTest() {
                 .isEqualTo(DoSomethingProvider.name);
     }
 
+    @Test
+    void providerToMultipleNames() {
+        FeatureProvider provider = new InMemoryProvider(Collections.EMPTY_MAP);
```

Comment:
`InMemoryProvider` to have the behaviour of an eventing provider

Reply:
maybe worth adding also a `FeatureProvider nop = new NoOpProvider();` to `clientC` and verify that it is registered correctly

Reply:
@thisthat thanks, updated the test to have non eventing noop provider as well - https://github.com/open-feature/java-sdk/pull/725/files#diff-6d6f404f03467a33bd72fef14be09ac8e70503ab9d4288b01ab179aeda1bb88eR48-R64